rust/parser/src/lexer.rs

```diff
@@ -344,11 +344,11 @@ impl<'alloc> Lexer<'alloc> {
         // SourceCharacter elements. A code point in a keyword cannot be
```

Comment:
Please modify the comment at the top of `identifier_tail` to add examples of what kind of input is lexed as TerminalId::NameMaybeKeyword, as TerminalId::Yield, and as TerminalId::Name.